frontend/src/views/Transactions.vue

```diff
@@ -10,6 +10,27 @@
     :config="config"
     :rows="rows"
   >
+    <template
+      #filters(accounts)="{filter}"
+    >
+      <label
+        v-for="(option) in filter.options"
+        :key="option.label"
+        :class="filter.value && filter.value.id === option.value.id? 'bg-gray-300' : ''"
+        class="inline-flex items-center px-2 mr-1 bg-gray-100 rounded-xl cursor-pointer hover:bg-gray-300 select-none"
+      >
+        <input
+          v-model="filter.value"
+          :text="option.label"
+          :value="option.value"
+          class="hidden"
+          name="radio"
+          type="radio"
```

Comment:
Shouldnt the qccount filter also be a dropdown? A user could have many accounts

Reply:
I wanted to have a list like that :

![image](https://user-images.githubusercontent.com/794449/114295548-e3e0a900-9a73-11eb-9a64-ff6c6f979e9b.png)

It could also be a dropdown, but the drawback with the dropdown is that the users need to open it to see it.

Reply:
i agree, with 2 accounts this looks nice and most people wont have more than that. We could eventually add a dropdown for more than 2 accounts.